src/data/customer/__test__/background-job.test.js

```diff
@@ -0,0 +1,62 @@
+import { getTokenAsync } from '../../account/Auth';
+import { getLatestBackgroundJob } from '../Background-Job';
+import CONFIG from './Config';
+
+describe('Tests getLatestBackgroundJob function', async () => {
+  let token;
+
+  beforeAll(async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+    token = await getTokenAsync(
+      CONFIG.email,
+      CONFIG.password,
+      CONFIG.clientId,
+      CONFIG.clientSecret
+    );
+  });
+
+  it('should response object including 200 status when type is 1(Optimize Route)', async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+
+    // type 1 is optimized routes
+    const response = await getLatestBackgroundJob(1, token.accessToken);
+    expect('data' in response).toBeTruthy();
```

Comment:
@vinay3206 you should validate a bit more in depth and make sure required values in data are present

Reply:
@maartenhemmes updated